trace_point.rb

```diff
@@ -135,6 +135,22 @@ def self.trace(*events)
     Primitive.tracepoint_trace_s(events)
   end
 
+  # call-seq:
+  #   TracePoint.allow_reentrance
+  #
+  # In general, while TracePoint callback is running,
+  # other registerred callbacks are not called to avoid
+  # confusion by reentrace.
+  # This method allow the reentrace. This method should be
+  # used carefully, otherwize the callback can be easily called
+  # infinitely.
+  #
+  # If this method is called when the reentrace is allowed,
+  # it raises a RuntimeError.
```

Comment:
```suggestion
  # In general, while a TracePoint callback is running,
  # other registered callbacks are not called to avoid
  # confusion by reentrance.
  # This method allows the reentrance. This method should be
  # used carefully, otherwise the callback can be easily called
  # infinitely.
  #
  # If this method is called when the reentrance is already allowed,
  # it raises a RuntimeError.
```

Reply:
Thank you!